Frends.Powershell/PowerShellHost.cs

```diff
@@ -0,0 +1,238 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Diagnostics;
+using System.Globalization;
+using System.Linq;
+using System.Management.Automation;
+using System.Management.Automation.Host;
+using System.Security;
+
+namespace Frends.PowerShell
+{
+    internal class TaskUserInterface : PSHostUserInterface
+    {
+        public override string ReadLine()
+        {
+            return ""; // We cannot provide user input
+        }
+
+        public override SecureString ReadLineAsSecureString()
+        {
+            return new SecureString();
+        }
+
+        public override void Write(string value)
+        {
+            Trace.Write(value);
```

Comment:
Could these actually write to some buffer that could then be returned in the results? So you could see any verbose output for long scripts, to see what went wrong.
But yeah, that could be a separate issue for the future.

Reply:
Yeah